com.civicrm.hrjobroles/js/test/filters/option-group-filter_test.js

```diff
@@ -0,0 +1,58 @@
+define([
+  'common/angular',
+  'common/lodash',
+  'common/angularMocks',
+  'job-roles/filters/option-group-filter',
+], function(angular, _, moment) {
+  'use strict';
+
+  describe('getActiveValues', function () {
+    var $filter, locations, getActiveValues;
+
+    beforeEach(module('hrjobroles.filters'));
+    beforeEach(inject(['$filter', function (_$filter_) {
+      $filter = _$filter_;
+      getActiveValues = $filter('getActiveValues');
+    }]));
+
+    it('is defined', function () {
+      expect($filter('getActiveValues')).toBeDefined();
+    });
+
+    describe('when filtering locations, departments, region and lavels ', function () {
+      beforeEach(function () {
+        locations = {
+          "Headquarters": {
+            "id": "Headquarters",
```

Comment:
These are not really ids, i believe. Ids are supposed to be numbers, aren't they?